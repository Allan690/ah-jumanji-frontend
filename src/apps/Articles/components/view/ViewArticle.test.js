import React from 'react';
import Enzyme, {shallow} from 'enzyme/build';
import Adapter from "enzyme-adapter-react-16/build";

import ArticlePage from './ViewArticle';


Enzyme.configure({ adapter: new Adapter() });

describe('Article Page', () => {
  let articlePage;
  beforeEach(() => {
    articlePage = shallow(<ArticlePage />);
  });

  it('renders correctly: ', () => expect(articlePage.exists()).toEqual(true));
  it('renders props: ', () => {
    const props = shallow(<ArticlePage articles="Article" />);
    expect(props.instance().props.articles).toBe("Article");
  });
});