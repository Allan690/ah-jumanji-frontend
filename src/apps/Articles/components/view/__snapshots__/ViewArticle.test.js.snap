// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewPage article page should render correctly in "debug" mode 1`] = `
<div
  className="container auth-container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <Connect(EditorView)
        slug="Yes-is-me"
      />
      <Connect(Rating) />
    </div>
  </div>
</div>
`;
